src/js/components/awardv2/shared/additionalInfo/IdvPeriodOfPerformance.jsx

```diff
@@ -0,0 +1,92 @@
+/**
+ * IdvPeriodOfPerformance.jsx
+ * Created by Lizzie Salita 6/25/19
+ **/
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+
+const propTypes = {
+    accordionData: PropTypes.object,
+    globalToggle: PropTypes.bool
+};
+
+export default class IdvPeriodOfPerformance extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            open: false
+        };
+        this.handleClick = this.handleClick.bind(this);
+    }
+    componentDidUpdate(prevProps) {
+        if (this.props.globalToggle !== prevProps.globalToggle) {
+            this.globalOverride();
+        }
+    }
+    handleClick() {
+        this.setState({ open: !this.state.open });
+    }
+
+    globalOverride() {
+        this.setState({
+            open: this.props.globalToggle
+        });
+    }
+
+    render() {
+        return (
+            <div className={this.state.open ? 'accordion accordion_open' : 'accordion'}>
+                <div
+                    className="accordion__bar"
+                    tabIndex={0}
+                    role="button"
+                    onKeyPress={this.handleClick}
```

Comment:
This should only trigger when the keyCode is 32 (space) or 13 (enter), correct?

https://keycode.info/

Reply:
Yes, this should be fixed in the generic `Accordion` component too.